src/rapidfuzz/process_py.py

```diff
@@ -670,3 +670,108 @@ def cdist(
                 results[i, j] = score
 
     return results
+
+def cpdist(
+    queries: Collection[Sequence[Hashable] | None],
+    choices: Collection[Sequence[Hashable] | None],
+    *,
+    scorer: Callable[..., int | float] = ratio,
+    processor: Callable[..., Sequence[Hashable]] | None = None,
+    score_cutoff: int | float | None = None,
+    score_hint: int | float | None = None,
+    score_multiplier: int | float = 1,
+    dtype: np.dtype | None = None,
+    workers: int = 1,
+    scorer_kwargs: dict[str, Any] | None = None,
+) -> np.ndarray:
+    """
+    Compute pairwise distance/similarity between corresponding elements in the two collection of inputs.
+
+    Parameters
+    ----------
+    queries : Collection[Sequence[Hashable]]
+        list of all strings the queries
+    choices : Collection[Sequence[Hashable]]
+        list of all pairwise strings the query should be compared. Must be the same length as the queries.
+    scorer : Callable, optional
+        Optional callable that is used to calculate the matching score between
+        the query and each choice. This can be any of the scorers included in RapidFuzz
+        (both scorers that calculate the edit distance or the normalized edit distance), or
+        a custom function, which returns a normalized edit distance.
+        fuzz.ratio is used by default.
+    processor : Callable, optional
+        Optional callable that is used to preprocess the strings before
+        comparing them. Default is None, which deactivates this behaviour.
+    score_cutoff : Any, optional
+        Optional argument for a score threshold to be passed to the scorer.
+        Default is None, which deactivates this behaviour.
+    score_hint : Any, optional
+        Optional argument for an expected score to be passed to the scorer.
+        This is used to select a faster implementation. Default is None,
+        which deactivates this behaviour.
+    score_multiplier: Any, optional
+        Optional argument to multiply the calculated score with. This is applied as the final step,
+        so e.g. score_cutoff is applied on the unmodified score. This is mostly useful to map from
+        a floating point range to an integer to reduce the memory usage. Default is 1,
+        which deactivates this behaviour.
+    dtype : data-type, optional
+        The desired data-type for the result array.Depending on the scorer type the following
```

Comment:
There is a space missing in front of `Depending`. That is already an issue in the documentation of `cdist` where this was copied from.

Reply:
Fixed. I also fixed it for cdist. Hope it is okay with you :)